packages/client/src/store/interfaces.ts

```diff
@@ -0,0 +1,18 @@
+export type InitialUserType = {
```

Comment:
совершенно не критично, но можем договориться называть типы не с постфиксом Type, а с префиксом T, так короче будет

```suggestion
export type TInitialUser = {
```

Reply:
Done